setup.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
`aboslute_import` was added automatically in every file where a relative import was present (8 files).  I've removed it from those files, and the tests are passing (against my non-s3 site), so I think we can skip them.  I've made a note of this and will add to my post-modernize todo list as something to check.  I think this would be more of a concern if we were continuing to support python 2.6.

Reply:
I've since read a bit on absolute import, I think it's fine to keep it in. We don't rely on that behaviour. It used to be that Python searched in the current package before looking into sys.path when doing absolute imports. We don't rely on the behaviour, so it's not necessary to add these imports at the top. They introduce noise more than anything.